src/main/resources/wa-task-permissions-wa-wacasetype.dmn

```diff
@@ -320,6 +320,606 @@ else
+          <text>"challenged-access-legal-ops"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0cga4t5">
+          <text>"Read,Manage,Cancel"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_15soxq3">
+          <text>"LEGAL_OPERATIONS"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_11rr31d">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_01le5rw">
+          <text>1</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1tfidpu">
+          <text>false</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_0ql7lde">
+        <description>Challenged Access (admin) task permissions</description>
+        <inputEntry id="UnaryTests_0g418sz">
+          <text>"reviewSpecificAccessRequestLegalOps"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_17nck6j">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_17ugdpw">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_10thrlm">
+          <text>"challenged-access-admin"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0rxzy6k">
+          <text>"Read,Manage,Own,Cancel"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1o3mfa1">
+          <text>"ADMIN"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_16rgoio">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0rekk0i">
+          <text>1</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0kz9w30">
+          <text>false</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1r3nz8w">
+        <description>Challenged Access (judiciary) task permissions</description>
+        <inputEntry id="UnaryTests_0d5gtri">
+          <text>"reviewSpecificAccessRequestAdmin"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1s6x0zw">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0k3sd6u">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_13te783">
+          <text>"challenged-access-judiciary"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0uvy8ci">
+          <text>"Read,Manage,Execute,Cancel"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_16qfxjx">
+          <text>"JUDICIAL"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1x076cn">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1j29e58">
+          <text>1</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0bm793v">
+          <text>false</text>
+        </outputEntry>
+      </rule>
     </decisionTable>
```

Comment:
What about "specific" grant type? Will be needed when verifying different query scenarios.

Reply:
specific access label added for roles